src/Translator/Extractor/JMSTranslatorBundle/AdminExtractor.php

```diff
@@ -210,12 +210,12 @@ public function transChoice($id, $number, array $parameters = [], $domain = null
 
     public function setLocale($locale): void
     {
-        $this->translator->setLocale($locale);
+        // $this->translator does not have setLocale function
```

Comment:
This class is deprecated and will be remove, maybe it's better to remove it before trying to add compability.